sdk/docs/manually-written/sdk/tutorials/setup-assistant/installation.rst

```diff
@@ -136,6 +136,36 @@ Download Manually
 
 If you want to verify the SDK download for security purposes before installing, you can look at :doc:`our detailed instructions for manual download and installation <manual-download>`.
 
+Troubleshooting
+***************
+
+The Daml installation requires around 1.5 GB of disk space. If there is not enough space on disk, the installer outputs the error:
+``
+resource exhausted (No space left on device)
+``
+
+In order to fix this problem, one needs to ensure sufficient disk space overall, and in particuloar that the user's home directory and the temporary
+files directory have sufficient space (i.e., in the order of 2GiB each):
+
+* On Windows, one can find the user's home directory in the `%USERPROFILE%` environment variable, and the temporary files directory in the
+  ``%TEMP%`` environment variable
+* On Linux and MacOS, one can find the user's home directory in the ``HOME%`` environment variable, and the temporary files directory in the
+  ``TMPDIR`` environment variable 
+
+If the installer cannot ensure sufficient space in the the user's home directory and the temporary files directory, an alternative is to change
+the location of these directories before the installation.
+To do so, call the installer with ``DAML_HOME`` and ``TMPDIR`` environment variables pointing to directories that have sufficient space:
+
+* On Windows, refer to the :ref:`path-variables <path-variables>` section for setting the environment variables. Then run the installation as usual.
+* On Linus and MacOS, run in a terminal:
+ 
+.. code-block: bash
+  
+  DAML_HOME=<YOUR-TARGET-DIR> TMPDIR=<YOUR-TEMPDIR> curl -sSL https://get.daml.com/ | sh
```

Comment:
We should also mention that you'll need to have the same DAML_HOME set and add `$DAML_HOME/bin` to the path in all terminals you use in the future, probably in .bashrc. Up to you if you want to include this here or in path-variables.rst